sdks/java/container/license_scripts/pull_licenses_java.py

```diff
@@ -0,0 +1,186 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+A script to pull licenses/notices/source code for Java dependencies.
```

Comment:
In addition to pulling the licenses, can this script generate a CSV of Library name, link to license, license name (like Apache, LGPL, etc), and whether source is downloaded/included?

Reply:
We can do that by adding some more code to the scripts. I will add it.

Reply:
This is done.